contracts/RiskManagerV1.sol

```diff
@@ -57,8 +57,11 @@ interface ITBTCDepositToken {
 /// @dev This interface is meant to abstract the underlying signer bonds
 ///      swap strategy and make it interchangeable for the governance.
 interface ISignerBondsSwapStrategy {
```

Comment:
I would argue we no longer need this interface. `swapSignerBonds` is not called by other contract and every implementation could have its own signature for the swap/withdrawal function. For example, `SignerBondsEscrow`, instead of `swapSignerBonds` that is not swapping anything, would probably be good with just `withdraw`.

Reply:
On second thought, we could have this interface with e.g. `onSignerBondsPurchased` implemented empty in existing strategies and left for future, in case in a third strategy we decide to do the swap automatically, on purchase.

Reply:
https://github.com/keep-network/coverage-pools/pull/88/commits/fefb57fb4ce4b9d57fbd22ee4fa6e4708b2a0640